odh-jupyter-trash-cleanup/ui-tests/jupyter_server_test_config.py

```
@@ -0,0 +1,12 @@
+"""Server configuration for integration tests.
+
+!! Never use this configuration in production because it
+opens the server to the world and provides access to JupyterLab
+JavaScript objects through the global window variable.
+"""
+from jupyterlab.galata import configure_jupyter_server
+
+configure_jupyter_server(c)
```

Ruff (0.12.2)

Undefined name `c`

(F821)